bundlesize.config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "files": [
+      {
+        "path": "packages/lwc/dist/engine-dom/esm/es2017/engine-dom.js",
```

Comment:
Huh, I thought `bundlesize` already did minification, but it looks like it doesn't. In any case you're right, since even minification would not remove the `process.env.NODE_ENV` blocks.